examples/nginx-access-logs-streaming/README.md

```diff
@@ -0,0 +1,217 @@
+# Analyzing Nginx Access Logs using Timeplus Proton 
+In a [recent article](https://www.timeplus.com/post/log-stream-analysis), we walked through three different ways of using Timeplus Proton to ingest and analyse a log file in real-time.
```

Comment:
Better to replace all "article" in this README with "blog". Although it's a bit odd use blog in README, but most readers of this content will be viewing this on our blog. So it should be fine.

Reply:
I've replaced "article" with "blog (post)" or "post" as appropriate.